benchmarks/cheshire/test/benchmark.clj

```diff
@@ -51,6 +56,13 @@
     (bench/bench (core/decode (core/encode test-obj)) :verbose))
   (println "-------------------------------------"))
 
+(deftest t-bench-pretty
+  (println "------- PrettyPrint Benchmarks ------")
+  (println "........default pretty printer")
+  (bench/bench (core/encode test-obj {:pretty true}))
+  (println "........custom pretty printer")
+  (bench/bench (core/encode test-obj test-pretty-opts)))
```

Comment:
Ah, you're speaking of reusing it in the benchmarks? Totally misunderstood, thought you want to reuse it inside create. Yes that makes sense.